src/test/ui/numbers-arithmetic/saturating-float-casts.rs

```diff
@@ -1,15 +1,19 @@
 // run-pass
 // Tests saturating float->int casts. See u128-as-f32.rs for the opposite direction.
-// compile-flags: -Z saturating-float-casts
+//
+// Some of these tests come from a similar file in miri,
+// tests/run-pass/float.rs. They're just duplicated currently but we may want
+// to merge this in the future.
 
 #![feature(test, stmt_expr_attributes)]
+#![feature(track_caller)]
 #![deny(overflowing_literals)]
 extern crate test;
 
 use std::{f32, f64};
-use std::{u8, i8, u16, i16, u32, i32, u64, i64};
-#[cfg(not(target_os="emscripten"))]
-use std::{u128, i128};
+#[cfg(not(target_os = "emscripten"))]
+use std::{i128, u128};
+use std::{i16, i32, i64, i8, u16, u32, u64, u8};
 use test::black_box;
 
 macro_rules! test {
```

Comment:
Further down this test file has a comment "If and when saturating casts are adopted, this macro should be merged with test!()". Seems like that time is now. :D